isis/tests/TestCsmPlugin.cpp

```diff
@@ -215,7 +237,17 @@ std::string TestCsmPlugin::convertISDToModelState(
 std::string TestCsmPlugin::getModelNameFromModelState(
     const std::string& modelState,
     csm::WarningList* warnings) const {
-  return "TestModelNameFromModelState";
+  auto state = json::parse(modelState);
+  std::string name = state.value<std::string>("model_name", "");
+
+  if (name == "") {
+    csm::Error::ErrorType aErrorType = csm::Error::INVALID_SENSOR_MODEL_STATE;
+    std::string aMessage = "No model_name key in the model state object.";
+    std::string aFunction = "TestCsmPlugin::getModelNameFromModelState";
+    csm::Error csmErr(aErrorType, aMessage, aFunction);
+    throw(csmErr);
+  }
+  return name;
```

Comment:
This is not how we are embedding the model name in the state strings. You should instead return the contents of the state string before the first endline. Something like `modelState.substr(0, modelState.find("/n"))` should do it.

Reply:
You're right. I was conflating this with how I'd updated the json isds to include the model_name, which I then didn't end up even using, so I'll update this accordingly